docs/tutorials/educators/intro.ipynb

```diff
@@ -99,18 +99,7 @@
    "metadata": {
```

Comment:
<p>Maybe reword for clarity: "...is the <code>sample</code>  method. This returns a pandas DataFrame where each column is a parameter/measurement key and each row is a repetition."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='120'/>

Reply:
ack!